src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -73,8 +73,10 @@ export default function FormNav(props) {
     );
   }
 
-  const stepText = `Step ${current} of ${chapters.length}: ${chapterName}`;
   const showHeader = Math.abs(current - index) === 1;
+  const hideFormNavProgress =
+    formConfig.chapters[page.chapterKey]?.hideFormNavProgress;
+  const stepText = `Step ${current} of ${chapters.length}: ${chapterName}`;
```

Comment:
I could've just made `stepText` null here based on `hideFormNavProgress` flag's value, but I actually prefer having the show/hide logic explicitly shown right inside the markup below.
[Add a comment if you want to discuss my preference further.]

Reply:
Very nitpicky comment haha sorry - but can you add property checking safety for the whole string?  for example `formConfig?.chapters?[page?.chapterKey]?.hideFormNavProgress` .  I know its overkill, but I've been bitten by NPEs in production too many times.  It also looks like page can be null as mentioned in other parts of this code, so good to be safe.  And can you explicitly make this true or false rather than undefined or value?

Reply:
Done & pushed.  :)